src/common/libflux/connector_loop.c

```diff
@@ -87,6 +89,56 @@ static flux_msg_t *op_recv (void *impl, int flags)
     return msg;
 }
 
+static int op_getopt (void *impl, const char *option, void *val, size_t size)
+{
+    loop_ctx_t *c = impl;
+
+    if (option && streq (option, FLUX_OPT_TESTING_USERID)) {
```

Comment:
Just curious if there's a case for this function to be called with a NULL `option`, and if not should it just have the following at the top?
```c
if (option == NULL) {
    errno = EINVAL;
    return -1;
}
```

Oh I see this just matches the usage in the local connector, but I'm still unsure of the use case for a NULL option.

Reply:
I just copied over what was there in the `loopback_create()` implementation, but I'll have a look and see if there's a bit more cleanup we can do.

Reply:
Seems fine to me for now if you want to defer cleanup for later. Just something that caught my eye is all..